base/bucket_gocb.go

```diff
@@ -1360,6 +1360,39 @@ func (bucket *CouchbaseBucketGoCB) DeleteWithXattr(k string, xattrKey string) er
 
 }
 
+func (bucket *CouchbaseBucketGoCB) GetXattr(k string, xattrKey string, xv interface{}) (casOut uint64, err error) {
```

Comment:
Unit tests for GetXattr should be added to bucket_gocb_test.go, and should at minimum cover:
- retrieval of existing xattr from existing doc
- retrieval of non-existent xattr from existing doc
- retrieval of non-existent xattr from non-existent doc
- retrieval of existing xattr from tombstoned doc
- retrieval of non-existent xattr from tombstoned doc